app-alternatives/cpio/cpio-0.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="CPIO symlink"
+HOMEPAGE="https://wiki.gentoo.org/wiki/No_homepage"
+SRC_URI=""
+S=${WORKDIR}
+
+LICENSE="metapackage"
```

Comment:
I wonder if `metapackage` is the best choice here. The packages install some trivial files like the manpage redirections.

Maybe some permissive license like `CC0-1.0` or `MIT` would be better?

Reply:
Well, I honestly don't think such a trivial and reproducible work of art is copyrightable but sure, will do.

Reply:
`app-alternatives/sh` technically installs symlink only but I've made it `CC0-1.0` too for consistency.